src/modules/polkadotSlotAuction/layout/components/NetworkSwitcher/NetworkSwitcherStyles.tsx

```diff
@@ -0,0 +1,36 @@
+import { Theme } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+
+export const useNetworkSwitcherStyles = makeStyles<Theme>(theme => ({
+  networkSwitcher: {
+    border: '1px solid rgba(255, 255, 255, 0.2)',
+    borderRadius: 24,
+    height: 42,
+    display: 'flex',
+    fontSize: 14,
+    lineHeight: '150%',
+    margin: '0 30px',
```

Comment:
for example 

```suggestion
    margin: theme.spacing(0, 3),
```
in design here should be not 30 but 24

Reply:
in my opinion 24 looks better

Reply:
They said to make 21, but after my advice to choose 24, they agreed with that